openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -33,12 +33,10 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
 
+        course_keys = []
         if options['all']:
-            # Have CourseOverview generate course overviews for all
-            # the courses in the system.
-            CourseOverview.get_all_courses(force_reseeding=True)
+            course_keys = [course.id for course in modulestore().get_courses()]
```

Comment:
Right, but I think the name is misleading and it doesn't make the requirement of a manual reseed clear. What if you added back the force_reseed option and had get_all_courses either run this logic or the cached logic, as it was, but without the history table? Then you could document the different cases in that method?